scripts/requirements.txt

```diff
@@ -1 +1,3 @@
-beautifulsoup4
+pandas
```

Comment:
I get it! Though pandas is so heavy duty (you have to install numpy right ...). I wonder if we can get away with something more lightweight.

Let's live with it for now!

Reply:
Agree. In this case I think is heavyweight for the computers but easy for humans to understand.